autoload/tsuquyomi/tsClient.vim

```diff
@@ -67,17 +70,57 @@ function! tsuquyomi#tsClient#startTss()
   return l:is_new
 endfunction
 
+function! s:startTssVim8()
+  if type(s:tsq['job']) == 8 && job_info(s:tsq['job']).status == 'run'
+    return 'existing'
+  endif
+  let l:cmd = substitute(tsuquyomi#config#tsscmd(), '\\', '\\\\', 'g')
+  try
+    let s:tsq['job'] = job_start(l:cmd)
+    let s:tsq['channel'] = job_getchannel(s:tsq['job'])
+
+    let out =  ch_readraw(s:tsq['channel'])
```

Comment:
I’ve updated `tsuquyomi` yesterday and found that vim now opens TS files with a couple seconds delay. After stepping through it with `echom` statements, I found that his `ch_readraw` call takes more than 2 seconds. 😶

I’m completely new to Vimscript, but it still looks like `out` variable is not used, nor `st` on the next line. So, just to try out, it I’ve deleted lines 81–89 and also the `try`/`catch` wrapping, and after that `tsuquyomi` still seems to work, and the 2 seconds startup delay seems to be gone. 🎉 🕺 

I’m sure there is more to the picture that I could see in my first long hour starring at this code, so, what do you guys think may be a reasonable approach to get rid of the startup delay? 🤔